go.sum

```diff
@@ -360,6 +372,9 @@ github.com/google/pprof v0.0.0-20201023163331-3e6fc7fc9c4c/go.mod h1:kpwsk12EmLe
 github.com/google/pprof v0.0.0-20201203190320-1bf35d6f28c2/go.mod h1:kpwsk12EmLew5upagYY7GY0pfYCcupk39gWOCRROcvE=
 github.com/google/pprof v0.0.0-20210122040257-d980be63207e/go.mod h1:kpwsk12EmLew5upagYY7GY0pfYCcupk39gWOCRROcvE=
 github.com/google/pprof v0.0.0-20210226084205-cbba55b83ad5/go.mod h1:kpwsk12EmLew5upagYY7GY0pfYCcupk39gWOCRROcvE=
+github.com/google/pprof v0.0.0-20210601050228-01bbb1931b22/go.mod h1:kpwsk12EmLew5upagYY7GY0pfYCcupk39gWOCRROcvE=
+github.com/google/pprof v0.0.0-20210609004039-a478d1d731e9/go.mod h1:kpwsk12EmLew5upagYY7GY0pfYCcupk39gWOCRROcvE=
+github.com/google/pprof v0.0.0-20210720184732-4bb14d4b1be1/go.mod h1:kpwsk12EmLew5upagYY7GY0pfYCcupk39gWOCRROcvE=
```

Comment:
I don't understand go deps well, so maybe a dumb question, but can we do a mod tidy and remove the duplicates?

Reply:
They're not duplicate because of the difference in version. I can't tell why these entries have the same hash for different versions. If you run:

```
$ go mod graph  |grep ' github.com/google/pprof'
```

you'll see some of these, and then even more of these. The [`go.sum`](https://go.dev/ref/mod#go-sum-files) file has every dependency, being direct or indirect computed by finding the [minimal version selection](https://go.dev/ref/mod#minimal-version-selection), and this is that output.

In other words, this has already been `go mod tidy`-ed.